src/api/Search/types.ts

```diff
@@ -171,6 +171,13 @@ export interface PhraseStatement {
   prefix: boolean;
 }
 
+/**
+ * This interface represents a search query that has been parsed by LKE backend.
+ * - The search query is divided into terms and pharses according the the search syntax specification.
+ *   - see https://doc.linkurious.com/user-manual/latest/search-syntax/
+ * - Please not that the terms generated by the backend are parsed but not tokenized.
+ *   - e.g: Search query: "email:support@linkurious.com" --> Parsed term: ["support@linkurious.com"] --> Tokenized term --> ["support", "linkurious", "com"]
```

Comment:
```suggestion
 *   - e.g: Search query: "email:support@linkurious.com" --> Parsed term: ["support@linkurious.com"]
 ```

i suspect this was the result of a copy&past, since this object is only for parsed and not for tokenized queries